shared/src/main/scala/tastyquery/reader/TreeUnpickler.scala

```diff
@@ -388,14 +412,17 @@ class TreeUnpickler(protected val reader: TastyReader, nameAtRef: NameTable) {
     if (reader.nextByte != SELFDEF) {
       reusable.EmptyValDef
     } else {
+      val curSpan = span
       reader.readByte()
       val name = readName
+      // TODO: Add span to TypeTree
```

Comment:
so I think the correct span the type tree's span

Reply:
Fixed